### PR TITLE
[nl] add exporting charts for nl

### PR DESCRIPTION
**Pull request by chejennifer**
- allow downloading svg and csv data for each chart in a tile

some bug fixes along the way:
- fix data fetch for bivariate tile (param was missing a plural)
- fix "download chart image" only showing up the second time the export button is clicked (here and place charts)

ezgif com-gif-maker

### CODE REVIEW

File: static/css/bq_modal.scss [hunk 1]
```diff
@@ -18,6 +18,14 @@
 
 @import "shared/modal.scss";
 
+// TODO (chejennifer): move this back to shared modal scss file.
```
Comment (beets) [inline static/css/bq_modal.scss:21]: super nit - could you add context for what's required to to this?

fwiw, i think it's ok for the modals to go very wide at the smaller screen sizes, even if the content is then surrounded by lots of padding.

File: static/js/components/tiles/bivariate_tile.tsx [hunk 2]
```diff
@@ -146,7 +157,7 @@ function getPopulationPromise(
         params: {
           parent_entity: placeDcid,
           child_type: enclosedPlaceType,
-          variable: variables,
+          variables: variables,
```
Comment (beets) [inline static/js/components/tiles/bivariate_tile.tsx:160]: thanks for the fix!

File: static/js/components/tiles/chart_tile.tsx [hunk 3]
```diff
@@ -28,29 +30,78 @@ interface ChartTileContainerProp {
...
+                event.preventDefault();
+                handleEmbed();
+              }}
+            >
+              Export
+            </a>
+          )}
+        </div>
+      </footer>
+      <ChartEmbed ref={embedModalElement} />
```
Comment (beets) [inline static/js/components/tiles/chart_tile.tsx:75]: should this also be wrapped with props.allowEmbed?

**Review by beets — APPROVED** [review]
very nice - thanks for the fixes and the tests!

**Comment by beets** [comment]
will merge this in first -- we can fix in followups!

File: static/css/bq_modal.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/bq_modal.scss:21]: this was a piece of code that was originally in the shared modal stylesheet and I think you had added it so I don't actually know the context of what it does. The reason I had to move it out of the shared modal stylesheet was that when I was trying to "@forward" and "@use" the different stylesheets, it was complaining about this piece of code ..

File: static/js/components/tiles/chart_tile.tsx (hunk 3 above)
Comment (chejennifer) [reply static/js/components/tiles/chart_tile.tsx:75]: yes, good call, will make a follow up PR for this

**Pull request merged**